tests/unit/package_managers/test_npm.py

```diff
@@ -977,6 +1081,7 @@ def test_resolve_npm_validation(
                     "purl": f"pkg:npm/foo@1.0.0?vcs_url={MOCK_REPO_VCS_URL}",
                     "bundled": False,
                     "dev": False,
+                    "missing_hash_in_file": [],
```

Comment:
Hmm, now that I'm looking at it, I think updating these tests would have made more sense. `test_get_sbom_components` doesn't really cover any real cases, these ones do. Removing the `integrity` in a few cases and keeping it in others should be enough, probably?